authors/apps/notifications/views.py

```diff
@@ -0,0 +1,46 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework import status
+from authors.apps.notifications.models import Notification, Notifier
+from rest_framework.permissions import IsAuthenticated
+
+class NotificationViews(APIView):
+    permission_classes = (IsAuthenticated,)
+    def get(self, request, username):
+        current_user = request.user
+        notifications = Notification.objects.filter(receiver=current_user)
+        all_notifications = []
+        print(notifications)
+        for notification in notifications:
+            notfy_details = {}
+            notfy_details['notification'] = notification.body
+            notfy_details['link'] = notification.link
```

Comment:
Rather than having to iterate through what could probably be hundreds of notifications, just to get a properly formatted dictionary, why don't you use serializers with a many=True argument?